klayout/drc/rule_decks/main.drc

```diff
@@ -0,0 +1,1072 @@
+
+      metal4        = metal4_drawn + metal4_dummy
+      
+      metal4_label  = get_polygons(46 , 10)
+      count   = metal4_label.count()
+      logger.info("metal4_label has %d polygons" % [count])
+      polygons_count  += count
+      
+      metal4_slot   = get_polygons(46 , 3 )
+      count   = metal4_slot.count()
+      logger.info("metal4_slot has %d polygons" % [count])
+      polygons_count  += count
+
+      metal4_blk    = get_polygons(46 , 5 )
+      count   = metal4_blk.count()
+      logger.info("metal4_blk has %d polygons" % [count])
+      polygons_count  += count
+      
+      top_via       = via3
+      topmin1_via   = via2
+      top_metal     = metal4
+      topmin1_metal = metal3
+    else
+      metal4_drawn  = get_polygons(46 , 0 )
+      count   = metal4_drawn.count()
+      logger.info("metal4_drawn has %d polygons" % [count])
+      polygons_count  += count
+
+      metal4_dummy  = get_polygons(46 , 4 )
+      count   = metal4_dummy.count()
+      logger.info("metal4_dummy has %d polygons" % [count])
+      polygons_count  += count
+
+      metal4        = metal4_drawn + metal4_dummy
+
+      metal4_label  = get_polygons(46 , 10)
+      count   = metal4_label.count()
+      logger.info("metal4_label has %d polygons" % [count])
+      polygons_count  += count
+
+      metal4_slot   = get_polygons(46 , 3 )
+      count   = metal4_slot.count()
+      logger.info("metal4_slot has %d polygons" % [count])
+      polygons_count  += count
+
+      metal4_blk    = get_polygons(46 , 5 )
+      count   = metal4_blk.count()
+      logger.info("metal4_blk has %d polygons" % [count])
+      polygons_count  += count
+
+      via4          = get_polygons(41 , 0 )
+      count   = via4.count()
+      logger.info("via4 has %d polygons" % [count])
+      polygons_count  += count
+
+      if METAL_LEVEL == "5LM"
+        metal5_drawn  = get_polygons(81 , 0 )
+        count   = metal5_drawn.count()
+        logger.info("metal5_drawn has %d polygons" % [count])
+        polygons_count  += count
+
+        metal5_dummy  = get_polygons(81 , 4 )
+        count   = metal5_dummy.count()
+        logger.info("metal5_dummy has %d polygons" % [count])
+        polygons_count  += count
+
+        metal5        = metal5_drawn + metal5_dummy
+
+        metal5_label  = get_polygons(81 , 10)
+        count   = metal5_label.count()
+        logger.info("metal5_label has %d polygons" % [count])
+        polygons_count  += count
+      
+        metal5_slot   = get_polygons(81 , 3 )
+        count   = metal5_slot.count()
+        logger.info("metal5_slot has %d polygons" % [count])
+        polygons_count  += count
+
+        metal5_blk    = get_polygons(81 , 5 )
+        count   = metal5_blk.count()
+        logger.info("metal5_blk has %d polygons" % [count])
+        polygons_count  += count
+
+        top_via       = via4
+        topmin1_via   = via3
+        top_metal     = metal5
+        topmin1_metal = metal4
+      else 
+        ## 6LM
```

Comment:
shouldn't that be 5LM?

Reply:
No that's correct, the else here is 6LM

Reply:
should we check for `6LM explicitly` and error if `else`?

Reply:
Done